specs/interop/predeploys.md

```diff
@@ -237,18 +237,34 @@ getters.
 |----------|----------------------------------------------|
 | Address  | `0x4200000000000000000000000000000000000015` |
 
-The `L1Block` contract is updated to include the set of allowed chains. The L1 Attributes transaction
-sets the set of allowed chains. The `L1Block` contract MUST provide a public getter to check if a particular
-chain is in the dependency set called `isInDependencySet(uint256)`. This function MUST return true when
-the chain's chain id is passed in as an argument.
+The `L1Block` contract MUST include method `setConfig(ConfigType, bytes)` for setting static values that are used
```

Comment:
I think that this should be in its own subsection

Reply:
If you made a subsection specific about the `setConfig` function then that would be great. It should include why it exists (reduces size of L1 attributes tx, reduces need to add specific one off functions)

After thinking through the upgrade process for this, I recommend that we don't include batcher hash or any of the other static config types yet. If we just do gas paying token + add + remove, I think it will be most simple for the upgrade. Otherwise we will have a really gnarly upgrade. This gives us the main benefit of not needing to include the full dep set into the L1 attributes tx every time

Reply:
The subsection should also include the definition of the `bytes` for each `ConfigType` option

Reply:
good idea!